index.html

```diff
@@ -2046,6 +2046,10 @@
 unrelated feature, by decreasing the quality of the service, or by trying to cajole, badger, or
 trick the [=person=] into opting back into the [=processing=].
 
+Note that because data is valuable, some sites allow users to pay for use of the site with data. If
+a user of this sort of site doesn't [=consent=] to exchange their data for use of the site, it's not
+retaliation to prevent them from using the site.
```

Comment:
Is there anything that then needs to be said about the nature of such deals?  https://en.wikipedia.org/wiki/Hobson's_choice perhaps?

Reply:
We can't make the blanket statement that denying some web resource to users who give some data consent is never retaliation. The "pay for the use of a site with data" covers a lot of different situations, from "fill in your email address to read a white paper" to "consent to surveillance on all your devices in order to get to your child's school announcements."

There are "controlled circulation" business models in publishing where a legit exchange of lead gen data for content takes place, but also cases where the power relationship between a site and user does not allow for actual consent.

Maybe better to say "some sites offer a free and fair exchange of appropriate information for original content they create or pay for the creation of, and such an exchange is not retaliation."